src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -852,13 +852,12 @@ public static void registerTags() {
         // @returns ElementTag(Boolean)
         // @group element checking
         // @description
-        // Returns whether the element contains any of a list of specified elements, case insensitive.
+        // Returns whether the element contains any of the specified elements, case insensitive.

```

Comment:
This meta change reduces specificity for no clean benefit. It specifies `a list of` to emphasize what's unique about this tag vs the other contains variants.

Reply:
Reverted